### ISSUE TITLE
Please add a VNDKLITE version

### ISSUE THREAD

**Issue by aarslan007** [realahnet/treble_evo#4]
Many phones require a VNDKLITE version, so it'd be awesome if you could please offer that. Thanks!

**Comment by realahnet** [comment]
> Many phones require a VNDKLITE version, so it'd be awesome if you could please offer that. Thanks!

you refer to many. does your phone require a vndklite version or does it work fine?

**Comment by aarslan007** [comment]
Yes, mine requires a VNDKLITE version, so my options are pretty limited at the moment.

**Comment by realahnet** [comment]
> Yes, mine requires a VNDKLITE version, so my options are pretty limited at the moment.

Alright i'll see for next builds

**Comment by aarslan007** [comment]
Awesome. Thank you so much!
